source/_components/conversation.markdown

```diff
@@ -20,6 +20,10 @@ To enable the conversion option in your installation, add the following to your
 ```yaml
 # Example configuration.yaml entry
 conversation:
+    boolean_test:
+      keywords: switch boolean
```

Comment:
I've also tweaked the config to change `keywords` to `sentence`. That should also be updated.